client/src/components/Home.vue

```diff
@@ -2,6 +2,18 @@
   <div>
     <Header />
 
+    <div class="search-bar">
```

Comment:
Once working, look into adding it as a separate component so we can reuse it in the admin dashboard as well.